client/sql/dataframe.go

```diff
@@ -31,6 +31,7 @@ type DataFrame interface {
 	Show(numRows int, truncate bool) error
 	Schema() (*StructType, error)
 	Collect() ([]Row, error)
+	Write() DataFrameWriter
```

Comment:
My suggestion is to add the basic documentation (even though it's replicated from OSS Spark)

Reply:
Good suggestion! Yes, I will update my PR with more comments/docs.

Reply:
Added document.